openedx/openedx-platform
# feat: allow update xblock fields without passing data  [FC-0062]

## Description

This PR modifies the update xblock fields endpoint, adding the capability to make partial updates without passing the `data` field.

## More information

Part of: 
- https://github.com/openedx/frontend-app-course-authoring/pull/1217

## Testing instructions

See https://github.com/openedx/frontend-app-course-authoring/pull/1217

---
Private ref: [FAL-3800](https://tasks.opencraft.com/browse/FAL-3800)
